test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +122,29 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
+		ns := f.Namespace.Name
+		c := f.ClientSet
+		client := c.CoreV1().PodTemplates(ns)
+		loopCount := int64(0)
+		for {
+			opts := metav1.ListOptions{}
+			opts.Limit = int64(rand.Int31n(numberOfTotalResources/10) + 1)
+			list, err := client.List(opts)
+			framework.ExpectNoError(err, "failed fetching PodTemplate list")
+
+			if loopCount > 0 && len(list.Continue) > 0 {
+				gomega.Expect(list.Continue[loopCount]).NotTo(gomega.Equal(list.Continue[loopCount-1]), "chunks are the same")
+			}
+			loopCount++
+			e2elog.Logf("Fetched %d chunks; Continue: %s; Items: %s", loopCount, list.Continue, fmt.Sprintf("%T", list.Items))
+
+			if len(list.Continue) == 0 || loopCount == int64(len(list.Continue)) {
```

Comment:
What does "loopCount == int64(len(list.Continue))" check?

Reply:
This is a check to make sure it's at the end of the chunking list

Reply:
I think "len(list.Continue) == 0" means it's at the end of chunking list.

I still don't get what "loopCount == int64(len(list.Continue))" indicates. Could you elaborate?

Reply:
The test has been updated in https://github.com/ii/kubernetes/commit/cee471ad3a972b37fd928aa6af1891d8323b75df and now no longer uses this section of code.